src/service/authentication.ts

```diff
@@ -0,0 +1,14 @@
+import firebase from 'firebase/app'
```

Comment:
We should declare the firebase modul, e.g. in a ```main.d.ts``` file, as there is a TS error on missing type declaration.